service/pay/helper/appdetail.go

```diff
@@ -0,0 +1,63 @@
+package helper
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/labring/sealos/service/pay/conf"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type PMDetail struct {
+	PayMethod     string   `bson:"payMethod"`
+	Currency      string   `bson:"currency"`
+	AmountOptions []string `bson:"amountOptions"`
+	ExchangeRate  float64  `bson:"exchangeRate"`
+	TaxRate       float64  `bson:"taxRate"`
+}
+
+func GetAppDetails(request *conf.Request) ([]PMDetail, error) {
+	appID := request.AppID
+	filter := bson.D{{"appID", appID}}
+	appColl := InitDB(os.Getenv(conf.DBURI), conf.Database, conf.AppColl)
```

Comment:
Do not initialize your database connections every time you receive your request and the handler gets called. First, it's huge resource waste since you don't reuse your database connections; Second, there will be noticeable response latency from the client side.